backend/.env.production.example

```
@@ -0,0 +1,6 @@
+# MCP — Production environment
+# Copy to .env.production and fill real credentials (never commit .env.production)
+DATABASE_URL="postgresql://tasktime:CHANGE_ME@72.56.7.218:5432/tasktime?schema=public"
+REDIS_URL="redis://72.56.7.218:6379"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)